### PR TITLE
Add history and comment endpoints

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -117,6 +119,71 @@ func (c *client) GetBug(id int) (*Bug, error) {
 	return bugs[0], nil
 }
 
+// GetBugComments retrieves the comments of a Bug from the server
+// https://bugzilla.readthedocs.io/en/latest/api/core/v1/comment.html#get-comments
+func (c *client) GetBugComments(id int, values *url.Values) ([]Comment, error) {
```
Comment (eparis) [inline client.go]: I don't think I like the idea of exposing url.Values as the public API.  What are we doing with this? Is this a leaky way to set comment_id or new_since?

File: client.go [hunk 2]
```diff
@@ -117,6 +119,71 @@ func (c *client) GetBug(id int) (*Bug, error) {
...
+	}
+	for _, comments := range parsedResponse.Bugs {
+		return comments.Comments, nil
+	}
+	return nil, nil
+}
+
+// GetBugHistory retrieves the history of a Bug from the server
+// https://bugzilla.readthedocs.io/en/latest/api/core/v1/bug.html#bug-history
+func (c *client) GetBugHistory(id int, values *url.Values) ([]History, error) {
```
Comment (eparis) [inline client.go]: Same issue here, probably should be a real golang struct which we turn into Values.

File: client.go (hunk 1 above)
Comment (sttts) [reply client.go]: new_since. Am happy to add optional option structs.

Comment (sttts) [reply client.go]: fixed (without option structs)

File: client.go (hunk 2 above)
Comment (sttts) [reply client.go]: fixed (without option structs)

**Comment by eparis** [comment]
I might break this API adding an options struct sometime. But it works.

**Pull request merged**